docs/curriculum-help.md

```diff
@@ -159,3 +159,33 @@ Using `__helpers.removeCssComments()`, `__helpers.removeHTMLComments()`, or `__h
 ### Removing Whitespace
 
 Using `__helpers.removeWhitespace()` allows you to pass the camper's code (through the `code` variable) to remove all whitespace.
+
+## AST-based Helpers
+
+These helpers need to be run in Python and they are built using the `ast` Python module.
+Tests that use them need to be in the format:
```

Comment:
```suggestion
These helpers need to be run in Python and tests that use them need to be in the format:
```
The fact they use `ast` not something you need to know to use the helpers, so I don't think it needs mentioning here.